pyserini/search/pysearch.py

```diff
@@ -254,6 +254,22 @@ def doc_by_field(self, field: str, q: str) -> str:
         """
         return Document(self.object.documentByField(JString(field), JString(q)))
 
+    def phrase_df(self, phrase: str) -> int:
+        """Constructs a phrase query and uses the TotalHitCountCollector to find the number of documents containing 
```

Comment:
I don't think there's a need to expose the implementation detail here? `TotalHitCountCollector`
Also be clear whether you expect an analyzed or unanalyzed phrase.
Any maybe we want a variant (optional arg) that does both?